### PR TITLE
remove extra chars in name patch

### PR DESCRIPTION
**Pull request by ThyWoof**

### CODE REVIEW

File: SolastaCommunityExpansion/Models/GameUiContext.cs [hunk 1]
```diff
@@ -21,28 +21,28 @@ internal static void Load()
...
-                {
-                    // Solasta original code disallows invalid filename chars and an additional list of illegal chars.
-                    // We're disallowing invalid filename chars only.
-                    // We're trimming whitespace from start only as per original method.
-                    // This allows the users to create a name with spaces inside, but also allows trailing space.
-                    textField.text = new string(
-                        textField.text
-                            .Where(n => !InvalidFilenameChars.Contains(n))
-                            .ToArray()).TrimStart();
+        //    public static bool Invoke(TMP_InputField textField)
```
Comment (PhilPJL) [inline SolastaCommunityExpansion/Models/GameUiContext.cs:28]: delete all the commented out code

Comment (PhilPJL) [reply SolastaCommunityExpansion/Models/GameUiContext.cs:28]: trailing space isn't a big deal

**Review by PhilPJL — APPROVED** [review]

**Review by PhilPJL — CHANGES_REQUESTED** [review]
Not sure we can remove - need to check

**Comment by ThyWoof** [comment]
Ok. Will leave for you to decide and merge.

**Pull request closed without merging**